packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -332,7 +332,7 @@ describe('Dropdown', () => {
       expect(titleElement.textContent).toEqual('4');
     });
 
-    it('Shows correct tooltip with and without title prop specified', () => {
+    it('Shows correct tooltip only with title prop specified', () => {
```

Comment:
Changing the behavior of existing unit tests like this is usually tricky because they usually have a reason for asserting a certain behavior... 

And so this would either be introducing a regression or a breaking change and hence we need to be extra careful.

The specific regression here is in the other comment above.

Reply:
reverting this change